appsec-kit-demo-v24-spring/src/main/java/com/vaadin/appsec/demo/v24spring/Application.java

```diff
@@ -8,23 +8,32 @@
  */
 package com.vaadin.appsec.demo.v24spring;
 
-import com.vaadin.flow.component.page.AppShellConfigurator;
-import com.vaadin.flow.theme.Theme;
+import java.nio.file.Paths;
+
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import com.vaadin.appsec.backend.AppSecConfiguration;
+import com.vaadin.appsec.backend.AppSecService;
+import com.vaadin.flow.component.page.AppShellConfigurator;
+import com.vaadin.flow.theme.Theme;
+
 /**
- * The entry point of the Spring Boot application.
- *
- * Use the @PWA annotation make the application installable on phones, tablets
- * and some desktop browsers.
- *
+ * The entry point of the Spring Boot application. Use the @PWA annotation make
+ * the application installable on phones, tablets and some desktop browsers.
  */
 @SpringBootApplication
 @Theme(value = "appsec-kit-demo-v24-spring")
 public class Application implements AppShellConfigurator {
 
     public static void main(String[] args) {
+        String userDir = System.getProperty("user.dir");
+        String packageJsonFilePath = userDir
+                + "/appsec-kit-demo-v24-spring/package.json";
```

Comment:
Do we still need the `package.json` file in the demo for V24, now that we get the SBOM from platform?

Reply:
We need to use the local `package.json` solution because when we get the platform SBOM we use this URL
https://github.com/vaadin/platform/releases/download/24.2-SNAPSHOT/Software.Bill.Of.Materials.json
to get it which contains the `flow-server` version number and in the AppSec Kit project we use the `24.2-SNAPSHOT` version for which there is no SBOM stored in the platform.